lib/transaction/trace/index.js

```diff
@@ -41,6 +41,11 @@ function Trace(transaction) {
       'host.displayName',
       displayName
     )
+
+    this.root.addSpanAttribute(
```

Comment:
per conversations about root not being turned into span... this may be problematic / may not actually make it up in a span. i missed this earlier.

Reply:
Yeah, I'm going to look for spots where I added attributes to root segment and add to active segment instead.